packages/sale/truffle.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
I've slimmed the truffle config to just this. I didn't see much point in keeping the other networks.